docs/platforms/native/advanced-usage/container-environments/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Container Environments
+description: "How to use the Sentry Native SDK in container environments."
+sidebar_order: 2000
+---
+
+## Database-path on a mounted volume
+The Sentry Native SDK uses a [database path](https://docs.sentry.io/platforms/native/configuration/options/#database-path) to store events and crash reports. When you are using a containerized environment, you may want to mount a volume to persist the database across container restarts to avoid losing this data.
```

Comment:
Should we have some more detailed explanation here as to what the `database_path` is used for?

Reply:
Like what, for example?

Reply:
As in, what kind of data the different backends store in the database path, and when these get picked up. As talked about, this might just warrant another advanced usage docs instead of me shoehorning it in here.